src/status_im/contexts/wallet/sheets/slippage_settings/view.cljs

```diff
@@ -73,7 +94,7 @@
              [quo/drawer-action
               (cond-> {:title    (str slippage "%")
                        :on-press #(on-select-slippage slippage)}
-                (= (str slippage) max-slippage) (assoc :state :selected))])
+                (and (= (str slippage) max-slippage) (not custom?)) (assoc :state :selected))])
```

Comment:
Why are we using `(str slippage)` to compare with `max-slippage` which is 100 (integer) ?

Reply:
`max-slippage` is string actually!